test/run_e2e_test.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+create_kind_cluster() {
+  # setup kind cluster
+  kind create cluster \
+    --retain \
+    --wait=1m \
+    -v=3
+
+  # debug cluster version
+  kubectl version
+}
+
+install_resources() {
+  kubectl apply -k test/test_resources
+}
```

Comment:
We don't need this, for setup we basically need to:
- Install CSI driver with snapshot-metadata service support along with the external-snapshot-metadata sidecar container
- A test app with volume mounted
We can setup this in a follow up PRs and focus on setting up cluster in this PR